packages/panzer/disc-fe/src/lof/Panzer_EpetraLinearObjFactory_decl.hpp

```diff
@@ -226,12 +226,14 @@ class EpetraLinearObjFactory : public LinearObjFactory<Traits>
 
    //! get the ghosted map from the matrix
    virtual const Teuchos::RCP<Epetra_Map> getGhostedMap() const;
+   virtual const Teuchos::RCP<Epetra_Map> getGhostedMap2() const;
```

Comment:
For the time being, there is now a version 2 of various functions, which will be used in conjunction with `EpetraVector_ReadOnly_GlobalEvaluationData`s.  If `EpetraLinearObjContainer`s are in use instead, the original versions of these routines will be used.  The goal would be to eventually rename all the version 2 routines to the original names.